### DISCUSSION TITLE
Background Blur

### DISCUSSION THREAD

**Discussion by d-muis** [realmazharhussain/gdm-settings#25] [Ideas]
Would you consider adding automatic blurring of the background? Having GDM look similar to the lock screen would be awesome, especially since a lot of us use the blur my shell extension. I know we can edit it manually but having it automated would make this app pretty much perfect in every regard.

**Comment by realmazharhussain** [comment]
I was going to. The feature used to be listed in (now deleted) "Planned Features" section of the README file. I tried using CSS filters to implement this feature. It didn't work. I'll have to use something like ImageMagick for this now.

I do plan on adding brightness and blur options to the GDM background image. But it may take some time (i.e. weeks or months) because it is not the current priority. Current priority is to provide the application in a universal packaging format (i.e. AppImage or Flatpak).

**Reply by d-muis** [reply]
awesome, it makes sense to prioritize porting to flatpak, great work btw ^^

**Reply by realmazharhussain** [reply]
Thanks
